### PR TITLE
[AR Number] Coverage Improvements - Ordinal Extractor, Number and Percentage Model spec updates

### PR DESCRIPTION
**Pull request by neudurgeshp**
**PR Includes**
- Number Definitions
- Specs
   - NumberModel
   - OrdinalModel
   - PercentModel

- Number Model Extractor
   - Total Cases 279: Extractor Passed - 139, Full Passed - 38, Skipped - 140
   - New Cases 135: Extractor Passed - 131, Full Passed - 30, Skipped - 4
   - Integer 48: Extraction Passed - 44, Full Passed - 7, Skipped - 4
   - Fraction 56: Extraction Passed - 56, Full Passed - 13, Skipped - 0
   - Decimal 9: Extraction Passed - 9
   - Power 11: Extraction Passed - 11, Full Passed - 0, Skipped - 0
   - Negative 11: Extraction Passed - 11, Full Passed - 11

- Ordinal Model Extractor
   - Total Test Cases 70: Extractor Passed - 58, Full Passed - 15, Skipped - 12
   - Existing Cases 31: Extractor Passed - 20, Full Passed - 18, Skipped - 11
   - New Cases 39: Extractor Passed - 38, Full Passed - 32, Skipped- 1

- Percent Model Extractor
   - Total Test Cases 27: Extractor Passed - 26, Full Passed - 14, Skipped - 1
   - Existing Cases 12: Extractor Passed - 11, Full Passed - 11, Skipped - 1
   - New Cases 15: Extractor Passed - 15, Full Passed - 3, Skipped - 0

** Below inputs corrected by language expert in NumberModel. Earlier inputs were not fraction.
- "Input": "الإجابة هي ناقص صفر فاصلة مائة وخمسة وثلاثون",
- "Input": "مائة على ألف وخمسة",

** Result (Value) changed for below inputs. Values were not correct.
- "Input": "جزء من واحد وعشرون اجزاء الملف فاضية",
- "Input": "مائة وثلاثة وثلثان",
- "Input": "-٢٣^١,١",
- "Input": "-٢٣--^١,١",

### CODE REVIEW

File: Specs/Number/Arabic/PercentModel.json [hunk 1]
```diff
@@ -210,7 +209,7 @@
   {
     "Input": "100%",
     "Comment": "PendingValidation",
-    "NotSupported": "dotnet",
+
```
Comment (tellarin) [inline Specs/Number/Arabic/PercentModel.json:212]: Please remove these empty lines.

File: Specs/Number/Arabic/NumberModel.json [hunk 2]
```diff
@@ -2026,27 +2026,27 @@
...
+          "value": "0.0476190476190476"
         },
         "Start": 0,
         "End": 17
       }
     ]
   },
   {
-    "Input": "مائة ألف أخماس",
+    "Input": "مائة على ألف وخمسة",
```
Comment (tellarin) [inline Specs/Number/Arabic/NumberModel.json:2037]: What was the previous input if not a fraction? Could we add that form to the specs for one of the other types?

File: Specs/Number/Arabic/NumberModel.json [hunk 3]
```diff
@@ -2136,19 +2136,18 @@
     ]
   },
   {
-    "Input": "الإجابة هي ناقص صفر فاصلة مائة وخمسة وثلاثون",
```
Comment (tellarin) [inline Specs/Number/Arabic/NumberModel.json:2139]: Same.

File: Specs/Number/Arabic/NumberModel.json (hunk 2 above)
Comment (neudurgeshp) [reply Specs/Number/Arabic/NumberModel.json:2037]: The input was intended to be fraction but we figured that out there was some discrepancy so we reviewed with Language Expert, and got corrected them. Older form of input was not valid native sentence. That was integer (but still not in correct form). 
We have such integer cases.

File: Specs/Number/Arabic/NumberModel.json (hunk 3 above)
Comment (neudurgeshp) [reply Specs/Number/Arabic/NumberModel.json:2139]: The input was intended to be fraction but we figured that out there was some discrepancy so we reviewed with Language Expert, and got corrected them. Older form of input was not valid native sentence. That was integer (but still not in correct form). 
We have such integer cases.

**Pull request merged**